src/stan/mcmc/hmc/nuts/base_nuts.hpp

```diff
@@ -152,14 +152,16 @@ namespace stan {
         names.push_back("stepsize__");
         names.push_back("treedepth__");
         names.push_back("n_leapfrog__");
-        names.push_back("n_divergent__");
+        names.push_back("divergent__");
+        names.push_back("E__");
```

Comment:
It's easy for a user who has read Michael's papers
to map energy__ to E__, but the reverse step is a little
more confusing.  On the other hand, just seeing energy__ isn't
going to help most of our users without going to Michael's
papers.  So I'd prefer energy__ to match the other params and
be a little less confusing, but I don't think it's a huge deal
either way.  Narrower column names do have advantages.

- Bob

> On Mar 13, 2016, at 10:51 PM, Daniel Lee <notifications@github.com> wrote:
> 
> In src/stan/mcmc/hmc/nuts/base_nuts.hpp:
> 
> > @@ -152,14 +152,16 @@ namespace stan {
> >          names.push_back("stepsize__");
> >          names.push_back("treedepth__");
> >          names.push_back("n_leapfrog__");
> > -        names.push_back("n_divergent__");
> > +        names.push_back("divergent__");
> > +        names.push_back("E__");
> 
> would you be ok with something slightly more verbose like energy__? I'm thinking about what users will see and ask. This is only going in the header of the output csv file, correct?
> 
> —
> Reply to this email directly or view it on GitHub.
>